src/views/Users.vue

```diff
@@ -1,19 +1,28 @@
 <template>
-  <PageViewer :pageTitle="pageTitle" :content="content" />
+  <user-list :users="users" />
 </template>
 <script>
-import PageViewer from "../components/PageViewer.vue";
+import UserList from "../components/Userlist/UserList.vue";
+import {fetchUsers} from "@/services/UserServices.js";
+import { showNotification } from "@/notification.js";
 
 export default {
   components: {
-    PageViewer,
+    UserList,
   },
   data() {
     return {
-      pageTitle: "Users Page",
-      content: "This is the users content",
-    };
+      users: [],
+    }
   },
+  async mounted() {
```

Comment:
I believe @dimitrov93 mentioned this already but we it is considered best practice to make calls to the backend in the created() method in stead of the mounted() -> https://stackoverflow.com/a/45814848/13062206. Is it causing problems to show the banner/notification when using created()?